alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -123,11 +123,32 @@ The mitigation depends on which resources are being rejected and why.
     secret key reference in the `ServiceMonitor` or `PodMonitor`
     configuration.
 
+When the alert is triggered by an object managed by a 3rd-party operator, it
+might not be possible to fix the root cause. The resolution will depend on the
+status of the operator:
+
+- The operator is a certified Red Hat operator.
+  - If the operator is installed in the `openshift-operators` namespace, it
+    should removed and installed in a different namespace because
+    `openshift-operators` might contain community operators which don't have
+    the same level of support.
+  - If the operator documentation requires adding the
+    `openshift.io/cluster-monitoring: "true"` label to the operator's namespace
+    during the installation, ensure that the label exists.
```

Comment:
Maybe
```suggestion
  - If the operator is deployed in another namespace than `openshift-operators` and the documentation requires adding the
    `openshift.io/cluster-monitoring: "true"` label to that namespace
    during the installation, ensure that the label exists.
```
to avoid any confusion with `openshift-operators`